### PR TITLE
[AMD][rocm-openmp-extras] - Updates for 4.3.1 and 4.5.0.

### PR DESCRIPTION
**Pull request by estewart08**
@haampie @srekolam @arjun-raj-kuppala

### CODE REVIEW

**Comment by srekolam** [comment]
@haampie @trws can we get the review for this ?

File: var/spack/repos/builtin/packages/rocm-openmp-extras/package.py [hunk 1]
```diff
@@ -180,7 +195,7 @@ def patch(self):
 
         # If not in a git repo the STRIP command will have an empty
         # argument. This is fixed in later versions.
-        if self.spec.version == Version('4.3.0'):
+        if self.spec.version >= Version('4.3.0'):
```
Comment (trws) [inline var/spack/repos/builtin/packages/rocm-openmp-extras/package.py]: If the strip is fixed in later versions, why is the check a greater than?

Comment (estewart08) [reply var/spack/repos/builtin/packages/rocm-openmp-extras/package.py]: Unfortunately, the small patch was not promoted to the release I originally thought. I expect this change to be in 5.0 and this condition will be updated once released.

**Comment by trws** [comment]
Makes sense, but in that case should we keep the comment or maybe make it a TODO so it doesn’t get missed?

**Comment by estewart08** [comment]
> Makes sense, but in that case should we keep the comment or maybe make it a TODO so it doesn’t get missed?

Changed comment to TODO.

**Review by trws — APPROVED** [review]

**Comment by srekolam** [comment]
can we get this reviewed and if okay,merged ?

File: var/spack/repos/builtin/packages/rocm-openmp-extras/package.py [hunk 2]
```diff
@@ -47,7 +53,9 @@
     "f3e19699ce4ac404f41ffe08ef4546e31e2e741d8deb403b5477659e054275d5",
     "f41f661425534b5cfb20e2c0efd9d0800609dc3876ee9c3f76f026d36abbfa35",
     "d6c3f3aaa289251a433d99d1cffe432812093089ae876a6863295a15066c1eaf",
-    "13d3525078fd1c569f7c8ea7fce439b04f6b03814bbe88600c08f95c788e7802"
+    "13d3525078fd1c569f7c8ea7fce439b04f6b03814bbe88600c08f95c788e7802",
+    "13d3525078fd1c569f7c8ea7fce439b04f6b03814bbe88600c08f95c788e7802",
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/rocm-openmp-extras/package.py:57]: Is this correct that shas are the same?

File: var/spack/repos/builtin/packages/rocm-openmp-extras/package.py [hunk 3]
```diff
@@ -56,10 +64,12 @@
     "51cc8a7c5943e1d9bc657fc9b9797f45e3ce6a4e544d3d3a967c7cd0185a0510",
     "91fdfadb94aa6afc1942124d0953ddc80c297fa75de1897fb42ac8e7dea51ab9",
     "31bbe70b51c259a54370d021ae63528a1740b5477a22412685afd14150fff6f4",
-    "ec6cc4a9c24f098496de3206714dafe9a714f06afacfe21d53a4e6344f9cb4c9"
+    "ec6cc4a9c24f098496de3206714dafe9a714f06afacfe21d53a4e6344f9cb4c9",
+    "ec6cc4a9c24f098496de3206714dafe9a714f06afacfe21d53a4e6344f9cb4c9",
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/rocm-openmp-extras/package.py:68]: Same question here

**Review by alalazo — COMMENTED** [review]
Just a couple of questions. Other than that @trws approved, so fine with me to merge.

File: var/spack/repos/builtin/packages/rocm-openmp-extras/package.py (hunk 2 above)
Comment (estewart08) [reply var/spack/repos/builtin/packages/rocm-openmp-extras/package.py:57]: Yes, the sha (13d3525) stayed the same for 4.3.0 to 4.3.1. 4.5.0 updated to 3d7277fd6.

File: var/spack/repos/builtin/packages/rocm-openmp-extras/package.py (hunk 3 above)
Comment (estewart08) [reply var/spack/repos/builtin/packages/rocm-openmp-extras/package.py:68]: Same answer as above. Sha stayed the same from 4.3.0 to 4.3.1.

**Pull request merged**